dotnet/SimpleSauce.Test/SauceSessionTests.cs

```diff
@@ -93,6 +93,32 @@ public void Start_WithChromeVersionSet_CreatesCorrectDriver()
 
             SauceSession.Options.ConfiguredChromeOptions.BrowserVersion.Should().Be("72");
         }
+        [TestMethod]
+        public void Start_WithSafari_SetsUsernameAndAccessKey()
+        {
+            SauceOptions = new SauceOptions();
+            SauceOptions.WithSafari();
+            SauceSession = new SauceSession(SauceOptions, _dummyDriver.Object);
+
+            SauceSession.Start();
+
+            var browserOptionsSetInSauceJson = SauceSession.Options.ConfiguredSafariOptions.ToString();
+            var browserOptionsSetInSauce = DeserializeToObject(browserOptionsSetInSauceJson);
+            AssertUsernameAndAccessKeyExist(browserOptionsSetInSauce);
+        }
+
+        private static Root DeserializeToObject(string browserOptions)
+        {
+            return JsonConvert.DeserializeObject<Root>(browserOptions);
+        }
+        private static void AssertUsernameAndAccessKeyExist(Root configuredSauceOptions)
+        {
+            configuredSauceOptions.SauceOptions.Username.Should().NotBeNullOrEmpty();
+            configuredSauceOptions.SauceOptions.AccessKey.Should().NotBeNullOrEmpty();
+        }
+        //TODO need a test that will validate that
+        //DriverImplementation.CreateRemoteWebDriver(Options.ConfiguredSafariOptions);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/app/SauceLabs/simple_sauce/pullRequest?prid=4626672)